README.md

```diff
@@ -1,124 +1,84 @@
-> **Warning**
->
-> Considered as unstable. But, most of the commonly used features are working as
-> expected.
+# Grm [![deno module](https://shield.deno.dev/x/grm)](https://deno.land/x/grm)
 
-<div align="center">
-
-# Grm
-
-[![deno module](https://shield.deno.dev/x/grm)](https://deno.land/x/grm)
-
-</div>
-
-Grm is an improved Deno port of [GramJS](https://github.com/gram-js/gramjs),
-written in TypeScript. GramJS is a popular MTProto API Telegram client library
-written in JavaScript for Node.js and browsers, with its core being based on
-[Telethon](https://github.com/LonamiWebs/Telethon).
-
-Deno Land module: https://deno.land/x/grm
-
-> What is Deno? https://deno.land
+> MTProto client for Deno ported from [GramJS](https://github.com/gram-js/gramjs).
 
 ## Documentation
 
-Consider following the documentation by GramJS maintainers.
+Currently, there is no documentation dedicated to Grm.
+You can use the GramJS documentation and API reference.
 
-- https://gram.js.org
-- https://painor.gitbook.io/gramjs
-- https://gram.js.org/beta
-
-See the [Quick Start](#quick-start) section for a minimal example to get
-started.
+- <https://gram.js.org>
+- <https://painor.gitbook.io/gramjs>
+- <https://gram.js.org/beta>
 
 ## Quick Start
 
-Here you'll learn how to obtain necessary information to create a Telegram
-application, authorize into your account and send yourself a message.
+Here you'll learn how to obtain necessary information to initialize the client, authorize your personal account and send yourself a message.
 
 First, you'll need to obtain an API ID and hash:
 
-1. Login into your [Telegram account](https://my.telegram.org).
-2. Then click "API development tools" and fill your application details (only
-   app title and short name are required).
-3. Finally, click "Create application".
-
-> **Warning**
->
-> Never share any API/authorization details, that will compromise your
-> application and account.
+1. Visit [my.telegram.org](https://my.telegram.org) and sign in.
+2. Click "API development tools" and fill your application details (only the app title and the short name are required).
+3. Click "Create application".
 
-When you've successfully created the application, replace the API ID and hash
-you got from Telegram in the following code.
+> Don't leak your API credentials.
+> They can't be revoked.
 
-`main.ts`
+Then use your API credentials with the following example code:
 
 ```ts
 import { StringSession, TelegramClient } from "https://deno.land/x/grm/mod.ts";
 
-// Login and create an application on https://my.telegram.org
-// to get values for API ID and API Hash.
-const apiId = 123456;
-const apiHash = "abcd1234";
+const appId = 123456;
+const appHash = "abcd1234";
```

Comment:
Isn't `apiId` and `apiHash` more better? In the App confiuration page of my.telegram.org it is actually "App api_id:" and "App api_hash:". So it is apiId and apiHash.

But since both are just different by one letter, both are okay for me.